tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,658 @@
+				err = UnMountDatastoreFromClusterOrHost(datastoreName, computeClusterPaths[1])
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+		}()
+
+		ginkgo.By("Verify datastore has permission for storage service account from supervisor cluster")
+		userPermission, err = verifyPermissionForWcpStorageUser(dataCenter, datastoreName, "", "", wcpServiceAccUsers[0], roleCnsDatastore)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(userPermission).To(gomega.BeTrue())
+		ginkgo.By("Verify datastore has permission for storage service account from supervisor cluster1 also")
+		userPermission, err = verifyPermissionForWcpStorageUser(dataCenter, datastoreName, "", "", wcpServiceAccUsers[1], roleCnsDatastore)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(userPermission).To(gomega.BeTrue())
+
+		ginkgo.By("Removing mounted shared datastore from supervisor cluster1")
+		err = UnMountDatastoreFromClusterOrHost(datastoreName, computeClusterPaths[1])
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		unMount = false
+		ginkgo.By("Verify datastore has permission for storage service account from supervisor cluster only")
+		userPermission, err = verifyPermissionForWcpStorageUser(dataCenter, datastoreName, "", "", wcpServiceAccUsers[0], roleCnsDatastore)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(userPermission).To(gomega.BeTrue())
+		ginkgo.By("Verify datastore has no permission for storage service account from supervisor cluster1")
+		userPermission, err = verifyPermissionForWcpStorageUser(dataCenter, datastoreName, "", "", wcpServiceAccUsers[1], roleCnsSearchAndSpbm)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(userPermission).To(gomega.BeTrue())
+	})
+
+	/* TESTCASE-6
+
+	Kill VC session from a service account and attempt CSI ops from the corresponding SVC
+
+	Steps:
+		1. Verify the SVC account is different for each cluster
+		2. Verify different VC sessions are created for each SVC
+		3. Kill VC sessions from svc1 and verify sessions from svc2 are not affected
+		4. Create sts on each of the clusters and verify that they are successful
+		5. Verify new VC sessions from svc1 are available.
+		6. Delete all the sts and pvcs which belong to them
+
+	*/
+
+	ginkgo.It("[csi-multi-svc] Kill VC session from a service account and attempt CSI ops from the corresponding SVC", ginkgo.Label(p0, multiSvc, stable), func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		var oldSessionIds [][]string
+		var newSessionIds [][]string
+		var err error
+		ginkgo.By("Getting VC session Id for both the supervisors")
+		for i := 0; i <= numberOfSvc-1; i++ {
+			sessionIDs := getVcSessionIDforSupervisor(supervisorIds[i])
+			oldSessionIds = append(oldSessionIds, sessionIDs)
+		}
+
+		ginkgo.By("Kill VC session from supervisor1")
+		killVcSessionIDs(oldSessionIds[0])
+		time.Sleep(90 * time.Second)
```

Comment:
remove static sleep

Reply:
removed static wait , handled it using polling